portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -0,0 +1,61 @@
+package com.google.sps.data;
+
+import java.util.Date;
+import com.google.appengine.api.datastore.Entity;
+
+
+/** Class containing users comments and time. */
```

Comment:
The comment is too implementation specific. Just say that the class represents a users comment.